test_requirements.txt

```
@@ -4,7 +4,7 @@ isort>=4.3.16, <5.0
 mock>=2.0.0,<3.0
 pre-commit>=1.17.0, <2.0.0
 pygraphviz>=1.5, <2.0
-pylint>=2.3.1, <3.0
+pylint>=2.3.1, <=3.0.0, !=2.5
```

Sorry! Can you constrain this as 
```suggestion
pylint>=2.3.1, <2.4.4
```
There is a security issue in 2.4.4, and a ci issue in 2.5!

---

ok! back to 2.4.4 then :-)